core/flyout.js

```diff
@@ -745,7 +745,8 @@ Blockly.Flyout.prototype.layout_ = function(contents, gaps) {
       if (this.horizontalLayout_) {
         cursorX += tab;
       }
-      block.moveBy((this.horizontalLayout_ && this.RTL) ? -cursorX : cursorX,
+      block.moveBy((this.horizontalLayout_ && this.RTL) ?
+          cursorX + blockHW.width - tab : cursorX,
```

Comment:
Sure!  In #461 I started reversing the contents of the blocks array and laying them out in the new order.  As part of that I stopped needing to make cursorX negative for horizontal RTL.  

When I added the flyout button I made some small tweaks to this code.  Then when I merged in from develop, there was a conflict on this line but not the line that reversed the block order, so I got the worst of both worlds.  Here's the offending commit: https://github.com/google/blockly/commit/097e5c4251fcfeee33f755c0f4174d3a6fa3bd0f#diff-7809e6395b6c09df1e557be57d043967